test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -69,7 +68,6 @@ public function testLeftOpenEndedRange() : void
     {
         $constraint = VersionConstraint::fromString('<1');
 
-        self::assertTrue($constraint->isSimpleRangeString());
```

Comment:
Why are `isSimpleRangeString` checks gone?

Reply:
mmm, because it is not longer necessary ? 
The only way version might not be parsed is due to wrong version format and in that case `Boundary::fromString()` will spit out `InvalidArgumentException` if I'm not mistaken.

Reply:
reverted this one